### DISCUSSION TITLE
Literate programming

### DISCUSSION THREAD

**Discussion by konnov** [informalsystems/quint#1066] [User stories]
We have been discussing for some time how it could be useful to generate Quint specifications from Markdown. This is especially useful for protocol designers. The workflow would look like follows:

 - Write prose in Markdown
 - Introduce Quint code snippets in the Markdown document
 - Generate Quint code from the Markdown document
 - Analyze the Quint specification, edit it
 - Update the Markdown document again

The very first prototype of this idea can be found in https://github.com/informalsystems/quint/blob/igor/cosmos-bank/examples/cosmos-bank/cosmos-bank.md

**Comment by shonfeder** [comment]
See #449
